applications/app/views/crosswordSearch.scala.html

```diff
@@ -9,7 +9,7 @@
             <header class="content__head tonal__head tonal__head--tone-news">
                 <div class="gs-container">
                     <div class="content__main-column">
-                        <div class="content__labels">
+                        <div class="content__labels content__labels--not-immersive">
```

Comment:
I wondered the same - it feels odd to have a class that describes what something isn't and not what it is. But maybe this is the most descriptive class (certainly more than --standard and mirrors --immersive).